infrastructure/nomad/playbooks/templates/jobs/l1-transactor.nomad.j2

```diff
@@ -99,8 +123,25 @@ job "{{ job.name }}" {
           {% for keystore_name in ns.keystore_path_passwd %}
             {{ ldelim() }} with secret "secret/data/mev-commit" {{ rdelim() }}
             mkdir -p "${TRANSACTOR_KEYSTORES_DIR}/{{ keystore_name }}" > /dev/null 2>&1
-            echo '{{ ldelim() }} .Data.data.{{ keystore_name }} {{ rdelim() }}' > "${TRANSACTOR_KEYSTORES_DIR}/{{ keystore_name }}/{{ ldelim() }} .Data.data.{{ keystore_name }}_filename {{ rdelim() }}"
+            TRANSACTON_KEYSTORE_FILE="${TRANSACTOR_KEYSTORES_DIR}/{{ keystore_name }}/{{ ldelim() }} .Data.data.{{ keystore_name }}_filename {{ rdelim() }}"
+            echo '{{ ldelim() }} .Data.data.{{ keystore_name }} {{ rdelim() }}' > "${TRANSACTON_KEYSTORE_FILE}"
             {{ ldelim() }} end {{ rdelim() }}
+
+            ADDRESS="$(cat "${TRANSACTON_KEYSTORE_FILE}" | jq -r '.address')"
```

Comment:
This is just distributing funds to accounts on the mock L1 right?

Reply:
Correct.